bin/phpcs

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
We now install phpcs as a phar file in the build directory as part of a composer install; this re-introduces the `./bin/phpcs` binary